saas_test/tests/test_saas_template.py

```diff
@@ -12,23 +13,31 @@ def setUp(self):
         super(TestSaasTemplate, self).setUp()
         self.saas_template = self.env['saas.template'].create({
             'template_modules_domain': 'mail',
-            'template_post_init': "self.env['mail.message'].create({'message_type': 'notification',}).message_format()"
+            'template_post_init': "self.env['mail.message'].create({'message_type': 'notification',}).message_format()",
+        })
+        self.saas_operator = self.env['saas.operator'].create({
+            'type': 'local',
         })
-        self.saas_operator = self.env['saas.operator'].create({})
         self.saas_template_operator = self.env['saas.template.operator'].create({
             'template_id': self.saas_template.id,
             'operator_id': self.saas_operator.id,
             'operator_db_name': 'test_db_name',
-            'to_rebuild': True,
-            'state': 'draft'
         })
 
     def test_template_operator(self):
         self.saas_template_operator.preparing_template_next()
+        job_id = self.env['queue.job'].search([('model_name', '=', 'saas.db')])
+        job = Job.load(self.env, job_id.uuid)
```

Comment:
Please assume that there could be other jobs with that model_name. So use the latest one